dataquality/loggers/data_logger/__init__.py

```diff
@@ -19,5 +23,7 @@
     "text_ner",
     "object_detection",
     "BaseGalileoDataLogger",
-    "seq2seq",
+    "base",  # Import needed for seq2seq subclass discovery
```

Comment:
Couldn't we just import directly in those files? Feels a bit weird to have `base` being exported.

Reply:
If we do need it, maybe makes sense to call it `seq2seq_base`? Just at the import-level would be fine IMO:
```suggestion
from dataquality.loggers.data_logger.seq2seq import base as seq2seq_base
...
    "seq2seq_base",  # Import needed for seq2seq subclass discovery
```

Reply:
Sounds good, i'll rename to base!